src/components/UserDetailSections/UserLoans/UserLoans.js

```diff
@@ -161,7 +161,7 @@ class UserLoans extends React.Component {
                 </Link>
                 {item.claimedReturnedCount > 0 &&
                   <span id="claimed-returned-count">
-                    <FormattedMessage id="ui-users.loans.numClaimedReturnedLoans" values={{ count: item.claimedReturnedCount }} />
+                    &nbsp;<FormattedMessage id="ui-users.loans.numClaimedReturnedLoans" values={{ count: item.claimedReturnedCount }} />
```

Comment:
Instead of `&nbsp;`, can we use `{' '}`?

Reply:
@zburke good point. let me update that and double check on that. Thank you.